skype/skype-ps/skype/Enable-CsUser.md

```diff
@@ -83,6 +83,13 @@ To do this, the Get-CsAdUser cmdlet is invoked, along with the Filter parameter.
 The filter {Enabled -ne $True} returns a collection of all the users who have not been enabled for Skype for Business Server.
 That collection is then piped to the Enable-CsUser cmdlet, which enables each account, assigning the user to the Registrar pool atl-cs-001.litwareinc.com and auto-generating a SIP address for each user.
 
+### -------------------------- Example 5 --------------------------
+```
+Enable-CsUser -Identity user@litwareinc.com -HostingProvider sipfed.online.lync.com -SipAddressType UserPrincipalName
+```
+
+Example 5 populates an existing Active Directory user in on-premises Skype for Business server who has been enabled for Skype for Business Online, or Microsoft Teams. The parameter -HostingProvider sipfed.online.lync.com specifies that the user is homed online, not on premises.
```

Comment:
```suggestion
Example 5 populates an existing Active Directory user in an on-premises Skype for Business server who has been enabled for Skype for Business Online, or Microsoft Teams. The parameter -HostingProvider sipfed.online.lync.com specifies that the user is homed online, not on-premises.
```

Reply:
I've re-written this Paragraph:

Example 5 populates an existing Active Directory user that is licensed for Skype for Business online, or Microsoft Teams but is not enabled in on-premises Skype for Business server. The parameter -HostingProvider sipfed.online.lync.com specifies that the user is homed online, not on-premises.

Are we able to merge the above paragraph changes?